docs/modpacks/playing_modpacks.md

```diff
@@ -61,3 +61,22 @@ Click the "Add a new Minecraft Core or Mod Pack" tile in the Start Menu, then cl
 ![Add a new Minecraft Core or Mod Pack Tile](../../static/img/tutorial/BakaXLAddCoreTile.png)
 
 Click "Continue" and drag the modpack file into BakaXL.
+
+### HMCL
+
+HMCL is a cross-platform Minecraft launcher which supports Mod Management, Game Customizing, Auto Installing (Forge, Fabric, Quilt, LiteLoader and OptiFine), Modpack Creating, UI Customization, and more.
+![HMCL](../../static/img/tutorial/Hmcl.png)
+
+#### Install through HMCL
+Installing Modrinth Modpack using HMCL is very simple. Go to the "Download" tab, select the Modpack, set the download source to "Modrinth," and then search for the Modpack you want. You can install and play it directly.
+
+![Hmcl search](../../static/img/tutorial/HmclSearch.png)
+
+#### Install manually
+
+This requires you to have a modpack file already downloaded.
+Go to the "All Instances" tab, click the "Install a Modpack", Click on "Import from local file" and then select your Modpack file. 
+If you have the direct link to the Modrinth Modpack, you can also choose "Download from the Internet" to import the instance.
+Alternatively, you can simply drag the Modpack file into the HMCL launcher interface.
```

Comment:
These will all run together as one paragraph. If you want them to be separate paragraphs, please have one newline between them.

Reply:
Thank you for pointing out my mistake. Line breaks have been added.